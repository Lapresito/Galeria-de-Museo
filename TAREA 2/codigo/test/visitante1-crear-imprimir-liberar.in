crearVisitante 19 Casimira Gonzalez 27
imprimirVisitante
liberarVisitante
crearVisitante 31 Esmeralda Rodriguez 30
imprimirVisitante
liberarVisitante
Fin
