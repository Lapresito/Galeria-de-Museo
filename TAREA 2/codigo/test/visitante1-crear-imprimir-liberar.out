Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creadox de forma exitosa.
2> Visitante Casimira Gonzalez
Id: 19
Edad: 27
3> Visitante liberado con exito.
4> Visitante creadox de forma exitosa.
5> Visitante Esmeralda Rodriguez
Id: 31
Edad: 30
6> Visitante liberado con exito.
7> Fin.
